client/components/Assignments/AssignmentItem/index.jsx

```diff
@@ -309,4 +255,17 @@ AssignmentItem.propTypes = {
     contentTypes: PropTypes.array,
     assignedDesk: PropTypes.object,
     contacts: PropTypes.object,
+    config: PropTypes.object,
 };
+
+const mapStateToProps = (state) => ({
+    config: state.config,
```

Comment:
You should be able to start using `import {appConfig} from 'appConfig';` without my PR being merged, as my PR removes the config from the redux store